packages/react/src/progress-indicator/ProgressIndicator.tsx

```diff
@@ -14,23 +14,36 @@ import { ProgressIndicatorList } from './ProgressIndicatorList';
 export type ProgressIndicatorProps = {
 	/** If the ProgressIndicator is placed on a page with 'bodyAlt' background, please set this to "bodyAlt". */
 	background?: 'body' | 'bodyAlt';
+	/** The list of items to display. */
 	items: ProgressIndicatorItem[];
+	/** If true, the `subTitle` above the list of items will not be rendered. */
+	hideSubtitle?: boolean;
 };
 
 export const ProgressIndicator = ({
 	background,
 	items,
+	hideSubtitle = false,
 }: ProgressIndicatorProps) => {
-	const stepsCompleted = items.filter((item) => item.status === 'done').length;
+	// The title in this component is not configurable
+	const title = 'Progress';
+
 	const totalSteps = items.length;
+	const stepsCompleted = items.filter((item) => item.status === 'done').length;
```

Comment:
Should we make this more efficient by only doing this if we're showing the subtitle? Something like
```suggestion
	const stepsCompleted = hideSubtitle ? 0 : items.filter((item) => item.status === 'done').length;
```